### DISCUSSION TITLE
Can't install on Code::Blocks

### DISCUSSION THREAD

**Discussion by AA1999** [dracula/dracula-theme#714] [Q&A] [answered]
I followed the instructions [here](https://draculatheme.com/codeblocks):
But the issue is, that the path `~/.codeblocks/cb_share_config` does not exist on my Linux.
I installed it from my repos
My distro is Arch
the folder `.codeblocks` simply does not exist in my home directory
I even created a new project to make sure it would be created

**Comment by luxonauta** [comment, marked as answer]
Hi 👋🏻, thank you for your interest in providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.

✅ [Clicking here](https://github.com/dracula/codeblocks/issues/new), you can open an Issue on it.
